pages/common/phpbu.md

```diff
@@ -0,0 +1,20 @@
+# phpbu
+
+> A backup utility framework for PHP.
+> More information: <https://phpbu.de>.
+
+- Run backups using the default "phpbu.xml" file:
+
+`phpbu`
+
+- Run backups using a specific configuration file:
+
+`phpbu --configuration={{path/to/configuration_file.xml}}`
+
+- Only run the specified backups:
+
+`phpbu --limit={{backup_task_name}}`
+
+- Don't actually create the backups:
```

Comment:
Maybe rephrase this? 
This description does not say what the command does. If fact, it says exactly the opposite.

Reply:
but the command itself says `simulate`. I think this description can be left as it is :/

Reply:
In my opinion you should not need to look at the example to understand the description. 
But if you guys feel that this is the best way i'll go with that.

Reply:
I agree with @schneiderl

Reply:
Or "Simulate actions phpbu would have permormed"... I am ok with both examples...